src/redux/reducers/experimentSettings/processingConfig/discardChangedQCFilters.js

```diff
@@ -1,9 +1,15 @@
 /* eslint-disable no-param-reassign */
-import produce from 'immer';
+import produce, { current } from 'immer';
 
 import initialState from '../initialState';
 
 const discardChangedQCFilters = produce((draft) => {
+  const originalProcessing = current(draft.originalProcessing);
+
+  Object.entries(originalProcessing).forEach(([key, value]) => {
+    draft.processing[key] = value;
```

Comment:
Do you need to loop through the entries can you not do `draft.processing = originalProcessing`?

Reply:
We have an entry called `meta` in the current processing that I don't want to disappear (which happens if I just assign `originalProcessing`)